start_your_career/style.css

```diff
@@ -1,11 +1,80 @@
 /* CSS Document */
-.headers {
+.body {
+    margin: 0px; 
+    background-color: #eceeec; 
+    font-family: Roboto, sans-serif;  
+}
+.header {
+    background-color: #373937;
+    padding: 10px;
+}
+.main_href {
+    line-height: 28px; 
+    font-family: Roboto, sans-serif;
+}
+.href {
+    color: white;
+    text-decoration: none;
+}
+.href_blog {
+    margin-right: 15px;
+    color: white;
```

Comment:
color: white;
 text-decoration: none;

Already exist in the .href

Reply:
Done.